infra/app/env-template/main.tf

```diff
@@ -246,7 +246,7 @@ module "staff" {
 }
 
 module "analytics_database" {
-  source        = "../../modules/database"
+  source        = "../../modules/database-rds"
```

Comment:
I think this naming may be unclear at some point in the future, as Aurora is a valid RDS target as well as serverless. 

Maybe database-serverless instead?